ivy_tests/test_ivy/test_frontends/test_numpy/test_random/test_functions.py

```diff
@@ -972,3 +972,60 @@ def test_numpy_triangular(
         right=right,
         size=size,
     )
+
+
+# noncentral_f - numpy
+@handle_frontend_test(
+    fn_tree="numpy.random.noncentral_f",
+    input_dtypes=helpers.get_dtypes("float"),
+    dfnum=st.floats(
+        allow_nan=False,
+        allow_infinity=False,
+        width=32,
+        min_value=1,
+        max_value=1000,
+        exclude_min=True,
+    ),
+    dfden=st.floats(
+        allow_nan=False,
+        allow_infinity=False,
+        width=32,
+        min_value=1,
+        max_value=1000,
+        exclude_min=True,
+    ),
+    nonc=st.floats(
+        allow_nan=False,
+        allow_infinity=False,
+        width=32,
+        min_value=0,
+        max_value=1000,
+    ),
+    size=helpers.get_shape(allow_none=False),
+)
+def test_numpy_noncentral_f(
+    input_dtypes,
+    size,
+    frontend,
+    test_flags,
+    backend_fw,
+    fn_tree,
+    on_device,
+    dfnum,
+    dfden,
+    nonc,
+):
+    test_flags.num_positional_args = 3
+    helpers.test_frontend_function(
+        input_dtypes=input_dtypes,
+        backend_to_test=backend_fw,
+        frontend=frontend,
+        test_flags=test_flags,
+        fn_tree=fn_tree,
+        test_values=False,
```

Comment:
Since we are not testing the values here, can you think of any *consistent* property that we can compare across frameworks for this?

Reply:
@hello-fri-end ,

How about the output shape? I have added a check to  check if the shape of the output is as expected: test_flags.check_output_shape = True

Reply:
`test_flags` does not have a `check_output_shape` attribute. You can take a look at [this](https://github.com/unifyai/ivy/blob/f5c75e99f3a414ced9b580ad0c3e815a002c1ce9/ivy_tests/test_ivy/test_functional/test_core/test_random.py#L95) test to understand how to manually test the values.